README.rst

```diff
@@ -56,8 +56,9 @@ Python code example
   from annoy import AnnoyIndex
   import random
 
-  f = 40
-  t = AnnoyIndex(f, 'angular')  # Length of item vector that will be indexed
+  f = 40  # Length of item vector that will be indexed
+
```

Comment:
Added this line to avoid visually mixing in the comments with the Annoy specific code